### PR TITLE
New function to fetch quotes (call endpoint with function GLOBAL_QUOTE)

### PR DESCRIPTION
**Pull request by volpatorafael**

### CODE REVIEW

File: test/e2e/stock-time-series/stock-time-series.test.ts [hunk 1]
```diff
@@ -155,3 +155,21 @@ describe('StockTimeSeries', () => {
     });
   });
 });
+
+describe('#quote', () => {
```
Comment (ricardo93borges) [inline test/e2e/stock-time-series/stock-time-series.test.ts:159]: please put this test inside the `describe('StockTimeSeries', () => { ` block, right after the `describe('#monthlyAdjusted', () => {` block, so the `quote `test can have access to the `api` object

File: src/stock-time-series/StockTimeSeries.ts [hunk 2]
```diff
@@ -225,4 +228,22 @@ export class StockTimeSeries extends Category {
       );
     }
   }
+
+  async quote(quoteDTO: QuoteDTO): Promise<QuoteResponse> {
```
Comment (ricardo93borges) [inline src/stock-time-series/StockTimeSeries.ts:232]: please add the unit tests for this method, the tests should be something similar to the `search `method unit tests

File: src/stock-time-series/StockTimeSeries.ts [hunk 3]
```diff
@@ -225,4 +228,22 @@ export class StockTimeSeries extends Category {
...
+    try {
```
Comment (ricardo93borges) [inline src/stock-time-series/StockTimeSeries.ts:233]: Also, update the README with this new function, please.

**Review by ricardo93borges — CHANGES_REQUESTED** [review]
The datatype description should be this one:

**datatype**: (optional) By default, datatype=json. Strings json and csv are accepted with the following specifications: json returns the quote data in JSON format; csv returns the quote data as a CSV (comma separated value) file.

File: README.md [hunk 4]
```diff
@@ -451,6 +452,38 @@ av.stockTimeSeries
...
+```js
+av.stockTimeSeries
+  .quote({ symbol: 'IBM' })
+  .then((data) => console.log(data));
+```
+
+**Parameters**
+
+1. **symbol**: The name of the equity of your choice. For example: symbol=IBM
+2. **datatype**: (optional) By default, datatype=json. Strings json and csv are accepted with the following specifications: json returns the intraday time series in JSON format; csv returns the time series as a CSV (comma separated value) file.
```
[the remark this answers is not in the document]
Comment (volpatorafael) [reply README.md]: updated

File: test/unit/stock-time-series/stock-time-series.test.ts [hunk 5]
```diff
@@ -738,4 +739,86 @@ describe('StockTimeSeries', () => {
...
+      api.get = jest.fn().mockResolvedValue({ data: givenQuoteResponse() });
+
+      const quoteDTO = {
+        symbol: 'IBM',
+        datatype: DataType.JSON,
+      };
+
+      const result = await stockTimeSeries.quote(quoteDTO);
+
+      expect(result).toEqual([
```
[the remark this answers is not in the document]
Comment (volpatorafael) [reply test/unit/stock-time-series/stock-time-series.test.ts]: Yes, the commit b6082b5e72806c6cce39147ac17faeaebbf9f27f fixed it

**Pull request merged**